src/components/details/index.js

```diff
@@ -42,16 +42,19 @@ const CarDetails = () => {
 
   return (
     <motion.div variants={detailsTransition} initial='out' animate='in' exit='out'>
+      <div className = 'disclaimer-container'>
+        <p className = 'disclaimer'>Based on averages over 5 years</p>
+      </div>
       <CarDetailsStyles className='detailsContainer'>
-        <div className='car-details'>
-          
+        <div className='carDetails'>
+
           <Card
             className='car-text'
```

Comment:
This class name needs to be updated, as well as the `car-img` className later in the file.

```suggestion
            className='carText'
```